server/views/pages/pop/view-appointment.njk

```diff
@@ -12,51 +13,34 @@
       text: "Back to Your appointments",
       href: "/pop/appointments"
     }) }}
-    <h1 class="govuk-heading-l govuk-!-margin-top-8 govuk-!-margin-bottom-1">View appointment</h1>
+    <h1 class="govuk-heading-l govuk-!-margin-top-6 govuk-!-margin-bottom-4">View upcoming appointment</h1>
+  <p class="govuk-body">If you have a problem attending an appointment, you need to <a class="govuk-link" href="">tell your probation practitioner</a> as soon as possible.</p>
   </div>
 </div>
 
-<div class="govuk-grid-row govuk-!-margin-top-6">
+<div class="govuk-grid-row govuk-!-margin-top-4">
   <div class="govuk-grid-column-full">
-    <div class="govuk-box govuk-box--rounded govuk-!-padding-4 border box-shadow">
-      <div class="govuk-grid-row govuk-!-margin-bottom-4">
-        <div class="govuk-grid-column-one-third">
-          <h3 class="govuk-heading-s">Meeting point</h3>
-        </div>
-        <div class="govuk-grid-column-two-thirds">
-          <p class="govuk-!-color-muted govuk-!-padding-0 govuk-!-margin-0">123 Garden Street, London SE1 7TH</p>
-        </div>
-      </div>
-      <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-4 govuk-!-color-light">
-
-      <div class="govuk-grid-row govuk-!-margin-bottom-2">
-        <div class="govuk-grid-column-one-third">
-          <h3 class="govuk-heading-s">Date</h3>
-        </div>
-        <div class="govuk-grid-column-two-thirds">
-          <p class="govuk-body">Saturday 15 March 2025</p>
-        </div>
-      </div>
-      <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-4 govuk-!-color-light">
-
-      <div class="govuk-grid-row govuk-!-margin-bottom-2">
-        <div class="govuk-grid-column-one-third">
-          <h3 class="govuk-heading-s">Time</h3>
-        </div>
-        <div class="govuk-grid-column-two-thirds">
-          <p class="govuk-body">9am - 5pm</p>
-        </div>
-      </div>
-      <hr class="govuk-section-break govuk-section-break--visible govuk-!-margin-bottom-4 govuk-!-color-light">
-
-      <div class="govuk-grid-row govuk-!-margin-top-2">
-        <div class="govuk-grid-column-full govuk-!-display-flex govuk-!-space-x-4">
-          <button class="govuk-button govuk-button--secondary govuk-!-margin-bottom-0 govuk-!-margin-right-2">
-            Cancel booking
-          </button>
-        </div>
-      </div>
-    </div>
+    {{ govukTable({
+      head: [],
+      rows: [
+        [
+          { text: "Type of appointment", classes: "govuk-!-font-weight-bold" },
```

Comment:
The bold styling here can be achieved with a table component attribute: `firstCellIsHeader: true,` before the head array attribute